src/palette/internal/palette/palettecreator.cpp

```diff
@@ -1099,11 +1099,15 @@ PalettePanel* PaletteCreator::newBracketsPalettePanel()
             { BracketType::LINE,   QT_TRANSLATE_NOOP("palette", "Line") } }
     };
 
-    for (auto type : types) {
+    static Staff bracketItemOwner(nullptr);
```

Comment:
I would have preferred to use a Staff belonging to gscore - but gscore doesn't have any staves and I don't know what the consequences of adding one might be.